CHANGELOG.md

```diff
@@ -86,6 +86,7 @@
 * [ENHANCEMENT] Querier: improve performance when shuffle sharding is enabled and the shard size is large. #4711
 * [ENHANCEMENT] Ingester: improve performance when Active Series Tracker is in use. #4717
 * [ENHANCEMENT] Store-gateway: optionally select `-blocks-storage.bucket-store.series-selection-strategy`, which can limit the impact of large posting lists (when many series share the same label name and value). #4667 #4695 #4698
+* [ENHANCEMENT] Querier: Put converted float histogram in chunkIterator.AtFloatHistogram into cache. #4684
```

Comment:
Is this user-visible?

Reply:
I don't fully understand the internal, but I believe there might be a small improvement with this fix although still might not be really noticeable to users.

Reply:
If we can rephrase this as something that user can notice, that would be great.

Reply:
Rephrased. It becomes a bit mouthful now :laughing: .